crates/astria-core/src/primitive/v1/asset.rs

```diff
@@ -68,6 +68,11 @@ impl Denom {
         self.prefix == prefix
     }
 
+    #[must_use]
+    pub fn is_prefixed_with(&self, prefix: &str) -> bool {
```

Comment:
Nit: I think one would ask "is something prefixed by?"
```suggestion
    pub fn is_prefixed_by(&self, prefix: &str) -> bool {
```